plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSplitSource.java

```diff
@@ -91,7 +92,12 @@ public HudiSplitSource(
                 queue,
                 new BoundedExecutor(executor, getSplitGeneratorParallelism(session)),
                 createSplitWeightProvider(session),
-                partitions);
+                partitions,
+                throwable -> {
+                    trinoException.compareAndSet(null, new TrinoException(GENERIC_INTERNAL_ERROR,
+                            "Failed to generate splits for " + table.getTableName(), throwable));
```

Comment:
nit: Unwrap.

Reply:
I am sorry , did not understand what needs to be changed here?

Reply:
unwrap means "keep it in a single line" :)

```
trinoException.compareAndSet(null, new TrinoException(GENERIC_INTERNAL_ERROR, "Failed to generate splits for " + table.getTableName(), throwable));
```

wrapping line here does not improve readability